src/ray/raylet/lib/java/org_ray_runtime_raylet_RayletClientImpl.cc

```diff
@@ -228,11 +231,12 @@ Java_org_ray_runtime_raylet_RayletClientImpl_nativeGenerateTaskId(
 
   TaskID task_id =
       ray::GenerateTaskId(driver_id.GetId(), parent_task_id.GetId(), parent_task_counter);
-  jbyteArray result = env->NewByteArray(sizeof(TaskID));
+  jbyteArray result = env->NewByteArray(kUniqueIDSize);
```

Comment:
use `kUniqueIDSize`->`task_id.size()` to keep constant style as following code.

Reply:
Sure, It should be modified in this pattern.